_posts/2017-12-15-bp-overview.markdown

```diff
@@ -49,72 +51,69 @@ When you look at this diagram, it seems simple but the reality is that not
 everyone has this knowledge. Even though it is basic, it provides a shared and
 common understanding, limits scope and deepens the context for analyses.
 
+<img src="/images/posts/bp-overview.png" alt="Activities">
+
 ### Which systems support the process?
 
 Let's pause going in further details for this process and talk about information systems.
 
-I identified the supporting systems for each activity. Following [ArchiMate][archimate]{:target="_blank"},
-I link two elements: business process with system software.
-
-<a href="/images/posts/ArchiMate-BP-Sys.png" target="_blank"><img src="/images/posts/ArchiMate-BP-Sys.png" alt="ArchiMate"></a>
-
 To determine an IT portfolio, I create a comprehensive list of the systems the company uses.
 When aiming for business-IT alignment, a key step is to link systems to
```

Comment:
Remove:

> When aiming for business-IT aligment,

Because it is another subject. Start from:

> A key step is ...

Reply:
Something you can add:

> This is important because software is the main driver of process execution nowadays. These links guide people to where they will find tools, data, triggers and reports to run the processes properly. It also helps to know what a person needs in terms of authentication and authorization to use those systems.